### PR TITLE
Add the English FAQ file for TiDB

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@shenli @c4pt0r @siddontang @ngaut PTAL.

File: TiDB_FAQ.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+TiDB is at your service if your applications require any of the following:
++ Horizontal scalability
++ Business continuity 
++ Strong consistency
++ Support for distributed ACID transactions
+
+## When not to use TiDB?
+
+TiDB is not a good choice if the number of the rows in your database table is less than a TB and there is no requirement for high availability, strong consistency and cross-datacenter replication.
```
Comment (shenli) [inline TiDB_FAQ.md]: less than 100GB

File: TiDB_FAQ.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+## How does TiDB manage user account?
+
+TiDB follows MySQL user account management system. You can create user accounts and authorize them. 
+
+You can use MySQL grammar to create user accounts. For example, you can create a user account by using the following statement:
+
+CREATE USER 'test'@'localhost' identified by '123';
+
+The user name of this account is "test; the password is “123" and this user can login from localhost only.
```
Comment (shenli) [inline TiDB_FAQ.md]: "test -> "test"

File: TiDB_FAQ.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+As your business grows, your database might face the following three bottlenecks:
+
++ Lack of storage resources which means that the disk space is not enough.
+
++ Lack of computing resources such as high CPU occupancy.
+
++ Not enough throughputs.
+
+You can grow TiDB as your business grows.
```
Comment (shenli) [inline TiDB_FAQ.md]: grow -> scale?

File: TiDB_FAQ.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
++ Atomicity: Each transaction in TiDB is "all or nothing": if one part of the transaction fails, then the entire transaction fails, and the database state is left unchanged. TiDB guarantees atomicity in each and every situation, including power failures, errors, and crashes. 
+
++ Consistency: TiDB ensures that any transaction brings the database from one valid state to another. Any data written to the ImDB database must be valid according to all defined rules, including constraints, cascades, triggers, and any combination thereof. 
+
++ Isolation: TiDB provides snapshot isolation (SI), snapshot isolation with lock (SQL statement: SELECT ... FOR UPDATE), and externally consistent reads and writes in distributed transactions.   
+
++ Durability: TiDB allows a collection of machines to work as a coherent group that can survive the failures of some of its members. So in TiDB, once a transaction has been committed, it will remain so, even in the event of power loss, crashes, or errors. 
+
+## How to set the lease parameter in TiDB?
```
Comment (shenli) [inline TiDB_FAQ.md]: How to choose

File: TiDB_FAQ.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+When the TiDB server is started, set the lease parameter in the command line. The value of the lease parameter impacts the Database Schema Changes (DDL) speed of the current session. In the testing environments, you can set the value to 1s for to speed up the testing cycle. But in the production environments, it is recommended to set the value to minutes (for example, 300s) to ensure the DDL safety.
```
Comment (shenli) [inline TiDB_FAQ.md]: The lease parameter is set from the command line when starting a TiDB server.

File: TiDB_FAQ.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+## Why is the DDL statement so slow when using TiDB?
+
+TiDB implements the online schema changes of [Google F1](http://research.google.com/pubs/pub41376.html). Generally, DDL is not a frequent operation. In case of DDL, the top priority of TiDB is to ensure the data consistency and business continuity. A complete DDL has 2 to 5 phases depending on the statement type. Each phase takes the time of 2 leases. Assuming one lease is 5 minutes, for a Drop Table statement which requires 2 phases, it takes 20 minutes (2 x 2 x 5 = 20). As what we have learned from Google F1, the DDL operation is handled by the database administrator (DBA) using special tools and it usually takes days.
```
Comment (shenli) [inline TiDB_FAQ.md]: online schema change algorithm

Comment (shenli) [inline TiDB_FAQ.md]: 5 minutes -> 1 minute. I have already updated the Chinese FAQ.

File: TiDB_FAQ.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+## What language can I use to work with TiDB?
```
Comment (shenli) [inline TiDB_FAQ.md]: language -> programming language

**Comment by queenypingcap** [comment]
@shenli @c4pt0r @siddontang @ngaut PTAL.

**Comment by ngaut** [comment]
PTAL  @c4pt0r

File: TiDB_FAQ.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+
+Yes. TiDB follows MySQL’s SQL grammar. You can use TiDB just you use MySQL.
+
+## Is TiDB based on MySQL?
+
+No. TiDB supports MySQL syntax and protocol, but it is a new open source database that is developed and maintained by PingCAP, Inc.
+
+## How does TiDB compare to traditional relational databases like Oracle and MySQL?
+
+TiDB scores in horizontal scalability while still maintains the traditional relation database features. You can easily increase the capacity or balance the load by adding more machines.
```
Comment (shenli) [inline TiDB_FAQ.md]: What's mean by 'score'?

File: TiDB_FAQ.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+
+## How does TiDB support SQL?
+
+TiDB is compatible with MySQL protocol. TiDB sever speaks MySQL protocol and follows MySQL’s SQL grammar. The MySQL protocol layer and the TiDB SQL layer within the TiDB server work together to translate MySQL requests to TiDB SQL plan.
+
+TiDB maps relational databases to Key-Value databases. It maps the relational table to Key-Value pairs and it also translate SQL statements to Key-Value operations. 
+
+## Can I use SQL statements to work with TiDB?
+
+Yes. TiDB follows MySQL’s SQL grammar. You can use TiDB just you use MySQL.
```
Comment (shenli) [inline TiDB_FAQ.md]: You can use TiDB just as using MySQL.

File: TiDB_FAQ.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+
+Inspired by Google F1 and Spanner, TiDB and TiKV adopt a highly-layered architecture. This architecture supports pluggable storage drivers and engines, which powers you to customize your database solutions based on your own business requirements. Meanwhile, this architecture makes it easy to debug, update, tune, and maintain. You won’t have to go through the entire system just to find and fix a bug in one module.
+
+## What does Placement Driver (PD) do?
+
+Placement Driver (PD) works as the cluster manager of TiDB. It manages the TiKV metadata and makes decisions for data placement and load balancing. PD periodically checks replication constraints to balance load and data automatically.
+
+## Is it easy to use TiDB?
+
+Yes, it is. When all the required services are started, you can use TiDB as easily as a MySQL server.  You can replace MySQL with TiDB to power your applications without changing a single line of code in most cases. You can also manage TiDB using the popular MySQL management tools.
```
Comment (c4pt0r) [inline TiDB_FAQ.md]: 'as easy as' or 'as easily as'? If I'm wrong, I'm sorry for my bad English...:)

File: TiDB_FAQ.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+
+## When to use TiDB?
+
+TiDB is at your service if your applications require any of the following:
++ Horizontal scalability
++ Business continuity
```
Comment (c4pt0r) [inline TiDB_FAQ.md]: High availability?

File: TiDB_FAQ.md [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
++ Strong consistency
++ Support for distributed ACID transactions
+
+## When not to use TiDB?
+
+TiDB is not a good choice if the number of the rows in your database table is less than 100GB and there is no requirement for high availability, strong consistency and cross-datacenter replication.
+
+## How does TiDB manage user account?
+
+TiDB follows MySQL user account management system. You can create user accounts and authorize them.
```
Comment (c4pt0r) [inline TiDB_FAQ.md]: 'authentication mechanism' is better than 'account management' , I think.

File: TiDB_FAQ.md [hunk 13]
```diff
@@ -0,0 +1,139 @@
...
+
+Yes. The transaction model in TiDB is inspired by Google’s Percolator, a paper published in 2006. It’s mainly a two-phase commit protocol with some practical optimizations. This model relies on a timestamp allocator to assign monotone increasing timestamp for each transaction, so the conflicts can be detected.  PD works as the timestamp allocator in a TiDB cluster.
+
+## Does TiDB have ACID semantics?
+
+Yes. ACID semantics are guaranteed in TiDB:
+
++ Atomicity: Each transaction in TiDB is "all or nothing": if one part of the transaction fails, then the entire transaction fails, and the database state is left unchanged. TiDB guarantees atomicity in each and every situation, including power failures, errors, and crashes. 
+
++ Consistency: TiDB ensures that any transaction brings the database from one valid state to another. Any data written to the ImDB database must be valid according to all defined rules, including constraints, cascades, triggers, and any combination thereof.
```
Comment (c4pt0r) [inline TiDB_FAQ.md]: ImDB database?

File: TiDB_FAQ.md [hunk 14]
```diff
@@ -0,0 +1,139 @@
...
+
+## How does TiDB scale?
+
+As your business grows, your database might face the following three bottlenecks:
+
++ Lack of storage resources which means that the disk space is not enough.
+
++ Lack of computing resources such as high CPU occupancy.
+
++ Not enough throughputs.
```
Comment (mrmiywj) [inline TiDB_FAQ.md]: `No enough` or `Not enough` ?

**Comment by c4pt0r** [comment]
leaved some comments, rest LGTM

File: TiDB_FAQ.md (hunk 14 above)
Comment (c4pt0r) [inline TiDB_FAQ.md]: I think 'No enough throughput' is correct.

File: TiDB_FAQ.md (hunk 8 above)
Comment (queenypingcap) [inline TiDB_FAQ.md]: it means to succeed in an activity or to achieve something:

File: TiDB_FAQ.md (hunk 14 above)
Comment (queenypingcap) [inline TiDB_FAQ.md]: "No" can be used as a noun, adj. and adv. While "not" is mostly used as an adv. When "No" is used as an adv., it is usually used in two situations:
1.The antonym of "Yes" to answer a question.
Are you going to Shanghai today?
No, I am not.
2. To embellish a comparative form: There is no better choice than fleeing the country.
However, when it comes to embellish word like "a", "any", "much", "many", "enough", etc,. we can only use "not".

**Pull request merged**